### DISCUSSION TITLE
In v2.0 how do you change the database settings after you've ran the installer?

### DISCUSSION THREAD

**Discussion by brolifen** [lithnet/access-manager#185] [Q&A] [answered]
After running the installer and running AMS for the first time it gets stuck on "Attempting to connect to the Access Manager Service". Looking at te service logs it seems like it can't connect to the database. My believe is that it's because I forgot to specify the instance name of the external SQL database. 

However now I have no idea on how to correct this. I've looked through some of the files, but I found no reference to the SQL connection string that the service is using. And since AMS is not starting, I can't change this in the GUI. 

Any idea where I can modify this connection string post installation? Perhaps a future suggestion is to validate the connection to the SQL database before continuing installation and in cases where the database hasn't been setup yet you could add the option to skip it for now in order to finish the installation anyway.

**Comment by brolifen** [comment, marked as answer]
Managed to find it and change it in the registry.

**Comment by ryannewington** [comment]
That's great feedback. Thanks @brolifen we will look at adding in a PowerShell cmdlet to change the SQL settings, and see what we can do about installer validation
